hazelcast/src/main/java/com/hazelcast/instance/GroupProperty.java

```diff
@@ -114,14 +114,7 @@
      */
     PERFORMANCE_METRICS_LEVEL("hazelcast.performance.metric.level", ProbeLevel.MANDATORY.name()),
 
-    /**
-     * The delay in seconds between monitor of the performance.
-     * <p/>
-     * The default is 30 seconds.
-     */
-    PERFORMANCE_MONITOR_DELAY_SECONDS("hazelcast.performance.monitor.delay.seconds", 30, SECONDS),
```

Comment:
Hmm, maybe we can create a section where we can put outdated properties and some kind of help text (reference to new property, explanation etc.). With this we could keep the existing properties clean and have a central place for the historical stuff.